ui/app/src/components/PublishDialog/PublishDialogForm.tsx

```diff
@@ -172,12 +121,11 @@ export type PublishFormProps = Pick<
 > & {
   publishingChannels: any[];
   disabled: boolean;
-  classes?: any;
```

Comment:
@rart I didn't add this one back because it was not being used in the component. Please let me know if you want me to implement.